check_source.py

```diff
@@ -57,7 +57,9 @@ def check_source_submission(self, source_project, source_package, source_revisio
                 # Check to see if other packages exist with the same source project
                 # which indicates that the project has already been used as devel.
                 if not self.is_devel_project(source_project, target_project):
-                    self.review_messages['declined'] = '%s is not a devel project of %s, submit the package to a devel project first' % (source_project, target_project)
+                    self.review_messages['declined'] = """%s is not a devel project of %s, submit the package to a devel project first. """ \
```

Comment:
Either use `"` with `\` or `()` set around the lines or one set of `"""`, but having `"""` for single line strings is pointless. Given the indentation issue and only two lines I would suggest the following.

```python
self.review_messages['declined'] = ('%s is not a devel project of %s, submit the package to a devel project first'
    'See https://en.opensuse.org/openSUSE:How_to_contribute_to_Factory#How_to_request_a_new_devel_project for details.'
    % (source_project, target_project))
```